docs/analytics-ga4/ga4-link-tracker.md

```diff
@@ -69,7 +69,11 @@ The `data-ga4-track-links-only` attribute ensures that only link clicks are trac
 </div>
 ```
 
-Note: the addition of the `data-ga4-set-indexes` data attribute on the parent element will allow the `index` and `index_total` properties to be set.
+Note: the addition of the `data-ga4-set-indexes` data attribute on the parent element will allow the `index` and `index_total` properties to be set. If links that we can amend directly appear alongside links generated by content items or publishing tools and we do not want to index the former, adding a `data-ga4-do-not-index` attribute will prevent those links from being indexed. For example:
+
+```html
+<a href="/link1" data-ga4-do-not-index>This link will not be indexed</a>
```

Comment:
I think it's worth putting this example within a `<div data-ga4-set-indexes>` element, and including a link that would get indexed, to make it clearer.

Reply:
Good point, will do 👍